qutebrowser/app.py

```diff
@@ -806,6 +806,18 @@ def __init__(self, args):
         self.launch_time = datetime.datetime.now()
         self.focusObjectChanged.connect(self.on_focus_object_changed)
 
+    def event(self, e):
+        if e.type() != QEvent.FileOpen:
+            return super(QApplication, self).event(e)
+
+        url = e.url()
+        log.misc.info("Got FileOpen event: %s" % url)
+        tabbed_browser = objreg.get('tabbed-browser', scope='window',
+                                    window='last-focused')
+        tabbed_browser.tabopen(url, related=False)
```

Comment:
You can probably extract some code from `process_pos_args` to do this and also handle `new_instance_open_target`.

Reply:
Done